Utilities/Development/mock_torch.js

```diff
@@ -175,7 +175,7 @@ const graphQlResponse = {
   }
 };
 
-webApp.post('*', (req, res) => {
+webApp.post('(.*)', (req, res) => {
```

Comment:
Why did it work before? Will this still work with the older version? Would this be safer for both versions instead:

```suggestion
webApp.post(/.*/, (req, res) => {
```

Reply:
Works with both the Debian 10 and Debian 11 versions of `node-express` so I've applied the suggestion. The reason why it broke was because of the version upgrade of the `node-path-to-regexp` package when upgrading from Debian 10 to Debian 11 - https://github.com/pillarjs/path-to-regexp#compatibility-with-express--4x